highlighting/highlighting-trad.json

```diff
@@ -8,6 +8,7 @@
         "blue": "\u0640*\u0627\u0640*\u0632\u0640*\u0631\u0640*\u0642\u0640*|blue",
         "brown": "\u0640*\u0628\u0640*\u0646\u0640*\u064a\u0640*|brown",
```

Comment:
Can we please make this uft8 and regular Arabic letters also this is so very hard to read!

Reply:
yes, but since these 2 files are generated by python, I will try to see if there is an option for that

Reply:
Yes there is, something like this probably: https://www.tutorialspoint.com/How-to-read-and-write-unicode-UTF-8-files-in-Python

Reply:
It's done !

Reply:
I still seem to see the unicode numbers but we can do that in a separate PR. Might be a tool issue.